src/server/common/constants.ts

```diff
@@ -194,6 +194,7 @@ export const constants = {
   buzz: {
     minChargeAmount: 500, // $5.00
     maxChargeAmount: 99999999, // $999,999.99
+    cutoffDate: new Date('2023-10-16T00:00:00.000Z'),
```

Comment:
```suggestion
    cutoffDate: new Date('2023-10-17T00:00:00.000Z'),
```
let's get some leeway since it's UTC